### ISSUE TITLE
RuntimeException when running app by installing debug APK 😱

### ISSUE THREAD

**Issue by luisburgos** [yellowme/youst-kotlin#30]
When launching app by using a debug APK `mx.yellowme.youst.dashboard.DashboardActivity` could not be found inside the `base.apk`

- [ ] Maybe Dashboard should be inside `app` module.

## Steps to reproduce

1. Generate debug APK.
2. Install it on any device.
3. Wait for the splash screen to finish executing main animation.

## Expected behavior
App should open Dashboard feature module properly.

## Actual behavior
App is crashing and throwing a RuntimeException.

## Screenshots
N/A

## Additional context
This error might be happening due the feature modules recently setup.

**Comment by luisburgos** [comment]
We found a [similar issue inside the Plaid repository](https://github.com/android/plaid/issues/344)

**Comment by kevingamboa17** [comment]
### Should we close this ticket? 

This "bug" is not a bug, is just part of the nature of Dynamic modules, we can't create normal `apk's` as we used to. 

We need to upload the generated `bundle` to the Play Console or [Share the app with an URL](https://support.google.com/googleplay/android-developer/answer/9303479), because all the modules will generate their APK, and the only way to share it is with Console itself.

![image](https://user-images.githubusercontent.com/16912316/81227131-887b8080-8fb1-11ea-8d4a-447c04aa809e.png)

**Comment by luisburgos** [comment]
You are totally right! Maybe we should add the proper Github Actions in order to build the app and deploy it to the play store? What do u think?

**Comment by kevingamboa17** [comment]
Yes! that's what i'm working on 🙌
